dev/com.ibm.websphere.appserver.features/visibility/public/mpReactiveStreams-1.0/com.ibm.websphere.appserver.mpReactiveStreams-1.0.feature

```diff
@@ -14,7 +14,8 @@ IBM-API-Package: \
   org.reactivestreams; type="stable";
 -features=\
   com.ibm.websphere.appserver.org.eclipse.microprofile.reactive.streams.operators-1.0, \
-  com.ibm.websphere.appserver.concurrent-1.0
+  com.ibm.websphere.appserver.concurrent-1.0, \
+  io.openliberty.mpCompatible-1.0
```

Comment:
we need to make this tolerate mpCompatible-4.0. Otherwise, MP Rest client 2.0 will not be able to work.

Reply:
done